README.md

```diff
@@ -50,6 +51,32 @@ scrape_configs:
 
 ```
 
+Result Codes in command_status:
+```
+    StatusOK       = 0
+    StatusWarning  = 1
+    StatusCritical = 2
+    StatusUnknown  = 3
+
+```
+Sample Alert Rule:
+```
+
+groups:
+- name: NRPE Host Load Status
+  rules:
+  - alert: HighLoad
+    expr: command_status{job="nrpe_check_load"} > 0
```

Comment:
https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 still applies